.github/workflows/zizmor.yml

```diff
@@ -0,0 +1,24 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
```

Comment:
<img width="958" height="346" alt="Screenshot 2025-07-31 at 1 46 54 PM" src="https://github.com/user-attachments/assets/33d6cd41-8cfd-431c-99bb-e8597bba5d36" />
This feels pretty out of place with our usual styling

Reply:
Yeah, that was copy paste from my template. I'll match the style here.

Reply:
(Waiting to push to see how far the builds get.)